tests/api_test.go

```diff
@@ -2109,6 +2115,105 @@ func TestProjectRunWebhookRedelivery(t *testing.T) {
 		expectedErr := util.NewRemoteError(util.ErrNotExist, util.WithRemoteErrorDetailedError(&util.RemoteDetailedError{Code: gwapierrors.ErrorCodeRunWebhookDeliveryDoesNotExist}))
 		assert.Error(t, err, expectedErr.Error())
 	})
+
+	t.Run("redelivery project run webhook delivery with deliverystatus = delivered and projectref = project path", func(t *testing.T) {
+		dir := t.TempDir()
```

Comment:
Instead of duplicating the same code by only changing the projectRef from id to path why don't you reuse the same code?

Reply:
@sgotti I should do it for all t.Run tests (in TestProjectRunWebhookRedelivery and TestProjectCommitStatusRedelivery) ?

Reply:
@alessandro-sorint The added test is a a copy of the previous one. Aren't the other tests quite different?